src/app/checkout/step-3/step-3.component.js

```diff
@@ -81,6 +81,7 @@ class Step3Controller{
   }
 
   submitOrder(){
+    this.submissionError = '';
```

Comment:
If submissionError isn't needed, you can `delete this.submissionError` instead of setting to empty string.